x-pack/solutions/search/plugins/search_indices/public/constants.ts

```diff
@@ -11,6 +11,7 @@ export enum QueryKeys {
   FetchSearchIndicesStatus = 'fetchSearchIndicesStatus',
   FetchUserStartPrivileges = 'fetchUserStartPrivileges',
   SearchDocuments = 'searchDocuments',
+  OnboardingData = 'onboardingData',
```

Comment:
currently this Query key `OnboardingData` isn't used in any other place. Why do we need this change?

Reply:
Used now with different approach